src/projects/apps/designer/typography.md

```diff
@@ -0,0 +1,24 @@
+# Typography
+
+These properties cover the font and it's styling used with this component.
+
+### Font Family
+Select a set of fonts to use in priority order (ie the first font will be the font that will be used unless it cannot be loaded, in which case the browser will try to use the next and so on). A font is one of the following:
+* **Google font**: loaded from the Google fonts library.
+* **Custom font**: a font from a file (or files) that you upload. Enter the name of your custom font into the font selector and press `enter`. Then attach the font file(s) (format should be `WOFF` or `WOFF2`) with the appropriate style and weight options and `Submit`. For example a font with `normal`, `italic`, `bold` and `bold italic` will be 4 separate rows.
```

Comment:
Is it possible to get a link to somewhere that explains font files etc?

Reply:
I have linked the format text to the MDN page for WOFF